src/config-schema.ts

```diff
@@ -1,43 +1,112 @@
-import { Type, validator } from '@openmrs/esm-framework';
+import { Type, validators } from '@openmrs/esm-framework';
 
 /**
- * This is the config schema. It expects a configuration object which
- * looks like this:
- *
- * ```json
- * { "casualGreeting": true, "whoToGreet": ["Mom"] }
- * ```
- *
- * In OpenMRS Microfrontends, all config parameters are optional. Thus,
- * all elements must have a reasonable default. A good default is one
- * that works well with the reference application.
- *
- * To understand the schema below, please read the configuration system
- * documentation:
- *   https://openmrs.github.io/openmrs-esm-core/#/main/config
- * Note especially the section "How do I make my module configurable?"
- *   https://openmrs.github.io/openmrs-esm-core/#/main/config?id=im-developing-an-esm-module-how-do-i-make-it-configurable
- * and the Schema Reference
- *   https://openmrs.github.io/openmrs-esm-core/#/main/config?id=schema-reference
+ * Represents the possible states of a billing line item
+ */
+export const BillingCondition = {
```

Comment:
If this is a type, better to use it as a Union, and not as an object and then getting the keys.

```suggestion
export type BillingConditionType = 'ORDER'|'INVOICED'|'NOT_INVOICED'|'FULLY_INVOICED'|'PARTIALLY_INVOICED'|'PAID'|'NOT_PAID'|'OVERDUE'|'NOT_OVERDUE'|'CANCELLED'
```

Reply:
I checked out the use of `BillingCondition` below, hence you can skip this comment.